woocommerce/woocommerce
# Photoswipe: Populate alt attribute for photoswipe galleries

### All Submissions:






### Changes proposed in this Pull Request:


Updates photoswipe and jquery zoom code to pull the alt attribute from 

Closes #25233.

### How to test the changes in this Pull Request:

1. Add a product gallery to a new or existing product.
2. Add images to the gallery that have alt text defined or add alt text to those images.
3. From the product page, select the "zoom" icon for one of the images in the gallery:
![image](https://user-images.githubusercontent.com/363749/86396411-7e9b9400-bc67-11ea-8b03-c7b0c4b8b25c.png)
4. Once photoswipe is displayed, inspect the image and verify that the alt attribute of the image is populated correctly:
![image](https://user-images.githubusercontent.com/363749/86396655-fa95dc00-bc67-11ea-9245-7b9012a18397.png)


### Other information:




### Changelog entry

> Accessibility: Adds alt attribute to photoswipe gallery images.
